Modules/ITS/src/ITSTrackTask.cxx

```diff
@@ -177,6 +179,22 @@ void ITSTrackTask::monitorData(o2::framework::ProcessingContext& ctx)
     hVerticesRof->Fill(nvtxROF);
   }
 
+  // DCAFitter2 class initialization  for the v0 part
+  using Vec3D = ROOT::Math::SVector<double, 3>; // this is a type of the fitted vertex
+  o2::vertexing::DCAFitter2 ft;
+  ft.setBz(-5.0);
+  ft.setPropagateToPCA(true);
+  ft.setMaxR(30);
+  ft.setMaxDZIni(0.1);
+  ft.setMaxDXYIni(0.1);
+  ft.setMinParamChange(1e-3);
+  ft.setMinRelChi2Change(0.9);
+  ft.setMaxChi2(10);
```

Comment:
Is it possible to add here comments about meaning of parameters you are using? It is not crucial, but would be very useful for code support